package.json

```diff
@@ -20,7 +20,7 @@
   "homepage": "https://github.com/brave/adblock-resources#readme",
   "devDependencies": {
     "adblock-rs": "^0.3.0",
-    "standard": "^16.0.4",
+    "semistandard": "^16.0.1",
```

Comment:
Lets not use Semi-Standard; having two similar but subtally different coding standards sounds like a real headache.  Happy to work with you on getitng something robust working that standardjs likes

Reply:
This is the error I get on Stable:
```Uncaught TypeError: (intermediate value)(intermediate value)(intermediate value)(...) is not a function
    at <anonymous>:80:1
(anonymous) @ VM8:80
```

and it seems to be because of the missing semicolon: https://st
[resources.json&DATfile.zip](https://github.com/brave/adblock-resources/files/8558280/resources.json.DATfile.zip)
ackoverflow.com/questions/42036349/uncaught-typeerror-intermediate-value-is-not-a-function

Reply:
My understanding is that it happens because we include multiple IIFs 
![image](https://user-images.githubusercontent.com/5284154/165182378-8485bc2a-b2b3-4436-82d4-bdac155141d3.png)

Reply:
I dont see the above code in the repo.   Where is that concatination happening?

this passes `standard` for me though, does this help:

```(self => {
  if (self.deAmpEnabled === true) {
    // Polyfill for requestIdleCallback
    // https://github.com/pladaria/requestidlecallback-polyfill/blob/master/index.js
    window.requestIdleCallback =
    window.requestIdleCallback ||
    function (callBack) {
      const start = Date.now()
      return setTimeout(function () {
        callBack({
          didTimeout: false,
          timeRemaining: function () {
            return Math.max(0, 50 - (Date.now() - start))
          }
        })
      }, 1)
    }
    const selector = 'a'
    const attr = 'jsaction'
    let timer
    const rmattr = () => {
      timer = undefined
      try {
        const nodes = document.querySelectorAll(selector)
        for (const node of nodes) {
          node.removeAttribute(attr)
        }
      } catch (ex) { }
    }
    const mutationHandler = (mutations) => {
      if (timer !== undefined) {
        return
      }
      let skip = true
      for (let i = 0; i < mutations.length && skip; i++) {
        const { type, addedNodes, removedNodes } = mutations[i]
        if (type === 'attributes') {
          skip = false
        }
        for (let j = 0; j < addedNodes.length && skip; j++) {
          if (addedNodes[j].nodeType === 1) {
            skip = false
            break
          }
        }
        for (let j = 0; j < removedNodes.length && skip; j++) {
          if (removedNodes[j].nodeType === 1) {
            skip = false
            break
          }
        }
      }
      if (skip) {
        return
      }
      timer = self.requestIdleCallback(rmattr, { timeout: 17 })
    }

    const start = () => {
      rmattr()
      const observer = new self.MutationObserver(mutationHandler)
      observer.observe(document, {
        attributes: true,
        attributeFilter: [attr],
        childList: true,
        subtree: true
      })
    }

    self.addEventListener('DOMContentLoaded', start, { once: true })
  }
})(window)
```

Reply:
Also, its dumb, but you can solve the problem its telling you about (the syntatic ambiguity of aligned IFF) if you just negate the call, a la
```(_ => {
  console.log('a')
})()

!(_ => {
  console.log('b')
})()```

though maybe standard wont like that easier?

But if thats no good, hows about just adding the semi colon and doing `// eslint-disable-line` for that line

Reply:
so `})(window); // eslint-disable-line`

Reply:
Disabling it for just that one line sounds like a good solution to me.